app/models/work.rb

```diff
@@ -605,7 +606,10 @@ def series_attributes=(attributes)
         errors.add(:base, ts("You can't add a work to that series."))
         return
       end
-      self.series << old_series unless (old_series.blank? || self.series.include?(old_series))
+      unless old_series.blank? || self.series.include?(old_series)
+        self.new_series = old_series 
+        self.serial_works.build(series: old_series)
+      end
       self.adjust_series_restriction
```

Comment:
I think this line can be removed, since the series' restricted status shouldn't change until the new `SerialWork` is saved to the database.